atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -252,7 +252,7 @@ def __init__(self, tasker, workflow, platforms, build_kwargs,
         :param build_kwargs: dict, keyword arguments for starting worker builds
         :param osbs_client_config: str, path to directory containing osbs.conf
         :param worker_build_image: str, the builder image to use for worker builds
-                                  (deprecated, use config_kwargs instead)
+                                  (deprecated)
```

Comment:
A better description would be `not used, image is inherited from the orchestrator)`

Reply:
ok

Reply:
@rcerven, do you plan to incorporate this suggestion?

Reply:
yes